Code/Engine/GameEngine/Effects/Wind/Implementation/SimpleWindWorldModule.cpp

```diff
@@ -19,6 +22,27 @@ ezSimpleWindWorldModule::~ezSimpleWindWorldModule() = default;
 
 ezVec3 ezSimpleWindWorldModule::GetWindAt(const ezVec3& vPosition) const
 {
+  if (auto pSpatial = GetWorld()->GetSpatialSystem())
```

Comment:
@C-Core This is really the core piece for implementing this feature. If you want to have support for the wind volume shapes in your custom wind system, you could just copy these lines.

Reply:
How about moving that into a static helper function or something like that?

Reply:
Well, it is quite likely that you'd do adjustments, e.g. to integrate the volume's output into your wind field simulation. Copy and paste would just be the simplest way to get started.